packages/client/webapp/docker/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM docker-hub.uncharted.software/nginx:1.23-alpine
+
+# update packages and add some basics
+RUN apk update
+RUN apk add --no-cache curl && apk add --no-cache bash && apk add --no-cache vim
+
+# remove default and add configuration
+COPY default.conf /etc/nginx/conf.d/
+
+# copy the web assets
+COPY ./dist /var/www/app
+
+# install wait-for-it
+ADD https://raw.githubusercontent.com/vishnubob/wait-for-it/master/wait-for-it.sh /usr/local/bin/wait-for-it.sh
+RUN chmod +x /usr/local/bin/wait-for-it.sh
```

Comment:
wait-for-it.sh is a pure bash script that will wait on the availability of a host and TCP port. It is useful for synchronizing the spin-up of interdependent services, such as linked docker containers.

Reply:
It's probably not needed with kubernetes. We could remove it if we need to, but it is useful if we want to run things without kube

Reply:
I agree that it could be useful.
I'm just wondering that because we are not linking to a specific version of the script but to the latest version that can be updated at any time, and we give it permission to execute on our machine.
This sounds like a potential source of trouble.

Maybe linking to the latest commit instead?
`https://raw.githubusercontent.com/vishnubob/wait-for-it/81b1373f17855a4dc21156cfe1694c31d7d1792e/wait-for-it.sh`

Reply:
ok with me